specification/eventgrid/resource-manager/readme.md

```diff
@@ -159,12 +159,22 @@ go:
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2018-09-preview
   - tag: package-2018-05-preview
   - tag: package-2018-01
   - tag: package-2017-09-preview
   - tag: package-2017-06-preview
 ```
 
+### Tag: package-2018-09-preview and go 
```

Comment:
Looking at the latest readme.me in the master, it looks like the Go configuration has been branched out to a separate file. It looks like you need to sync those changes in your branch and this change needs to happen in the GO specific readme file.